style.css

```diff
@@ -0,0 +1,610 @@
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  width: 76px;
+  height: 20px;
+  text-decoration: none;
+}
+
+#nav-items {
+  display: none !important;
+}
+
+.header-info-secion {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 100%;
+  background-color: #fff;
+  background-image: url(img/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  top: 2.875rem;
+  border-bottom-left-radius: 60px;
+  height: 44.88rem;
+}
+
+.greeting-hearder {
+  display: flex;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  align-items: center;
+  width: 327px;
+  height: 9.37rem;
+  margin: 20px;
+  color: #172b4d;
+  order: 0;
+}
+
+.greeting-text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  width: 327px;
+  height: 146px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 0 20px;
+  color: #344563;
+  order: 1;
+}
+
+.connect {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  text-decoration: none;
+  font-size: 16px;
+  font-weight: 500;
+  margin: 0 20px;
+  order: 2;
+  line-height: 24px;
+  color: #7f8cff;
+  flex-grow: 0;
+  overflow-x: hidden;
+}
+
+.twitter:hover,
+.linkedin:hover,
+.github:hover,
+.medium:hover,
+.angel:hover {
+  color: #505f79;
+  opacity: 0.5;
+}
+
+#social-icons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex: none;
+  flex-grow: 0;
+  list-style: none;
+  width: 10.636rem;
+  margin-left: -25px;
+  gap: 8px;
+  order: 3;
+}
+
+/* ----The Work Section */
+.work {
+  background: #e5e5e5;
+  display: grid;
+  grid-template-columns: 33.333%;
+  margin-top: 5rem;
+}
+
+.card-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  width: 327px;
+  height: 552px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 6px;
+}
+
+.project-img {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card-tittle {
+  width: 295px;
+  height: 36px;
+  margin-top: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.frame2-img {
+  width: 236px;
+  height: 32px;
+  padding: 0.5rem;
+  margin-top: -30px;
+  margin-bottom: 0;
+  margin-right: 0;
+  margin-left: -50px;
+}
+
+.description {
+  width: 295px;
+  height: 50px;
+  margin-top: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  flex-grow: 0;
+}
+
+.languages-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
```

Comment:
[OPTIONAL] - I can see so many repeated usage of `display: flex`. 

If you want to create a CSS rule for multiple elements, you can create a class and add it to all elements that need to be flexed. 👍💪

```css
.flex-row {
    display: flex;
    justify-content: center;
    align-content: center;
}

.flex-column {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
}
```